charts/managed-serviceaccount/values.yaml

```diff
@@ -8,3 +8,7 @@ replicas: 1
 
 featureGates:
   ephemeralIdentity: false
+
+# Name of the image pull secret in the manager namespace that agent deployment will use to pull the image
+#   uncomment and configure the next line if agent image is not publically accessible
+# agentImagePullSecret: ""
```

Comment:
nit: i think we dont need to comment this line

Reply:
uncommented `agentImagePullSecret: ""` line